python-backend/wolfram_client.py

```diff
@@ -60,11 +61,15 @@ def closed_form(expression: str) -> dict:
         """
         try:
             # Wolfram has a 200 character input limit
+            assert len(expression) == 200
```

Comment:
need to log this scenario for debugging purposes

Reply:
Should this be <= ? Just going off the comments from the last PR

Reply:
🤦🏻 multitasking is bad